tests/integration/test_integration.py

```diff
@@ -1,4 +1,15 @@
-"""Test the simplest proxy use scenario for smoke."""
+# -*- coding: utf-8 -*-
+"""
+    proxy.py
+    ~~~~~~~~
+    ⚡⚡⚡ Fast, Lightweight, Pluggable, TLS interception capable proxy server focused on
+    Network monitoring, controls & Application development, testing, debugging.
+
+    :copyright: (c) 2013-present by Abhinav Singh and contributors.
+    :license: BSD, see LICENSE for more details.
+
+    Test the simplest proxy use scenario for smoke.
```

Comment:
Per PEP 257, a docstring should start with a title up to 72 chars followed by an empty line and then, the rest of the prose.

Reply:
Totally.  I'll keep these changes and address them as part of doc-string.  Easier to fix via `awk`/`sed` in a separate branch :) Actually `Makefile` was broken for sometime.  I fixed it today which started running `check.py` and it caught all files without library header.  Had to add back in missing files.

Reply:
Created https://github.com/abhinavsingh/proxy.py/issues/776 for `PEP 257` related stuff